include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
I agree mutually dependent headers should be avoided. But, in this case, I'd make an exception to that, mostly because they are both configuration headers. We need the StandardCLibrary halfway the CppUTestConfiguration. Usually we would included the StandardC directly there and this wouldn't occur. I'd rather all the StandardC library includes in one place though that causes this problem.

If people don't need TestHarness but only StandardC, then I wouldn't want to force them to include TestHarness

So, for now, keeping it like this unless it would actually cause a problem.